common/source/docs/common-gps-for-alt.rst

```diff
@@ -0,0 +1,26 @@
+.. _common-gps-for-alt:
+
+==============================
+Using GPS for Altitude Sensing
+==============================
+
+Until version 4.1, ArduPilot required a barometer altitude sensor. Now, it is possible to boot an autopilot without barometer present, and use GPS (outdoor use) or other altitude sensor (rangefinder, range beacon) for indoor use. 
+
+.. warning:: Without a barometer as a backup sensor, loss of the primary altitude sensor can result in a crash if in an altitude holding mode or autonomous mode. Be prepared to switch to a non altitude holding mode in case of loss of primary sensor.
+
+Setup for using a non Baro Source for Altitude Measurement
+==========================================================
+
+If using EKF2:
+
+- Set the :ref:`EK2_ALT_SOURCE<EK2_ALT_SOURCE>` parameter to GPS.
```

Comment:
I think we should specify the actual number so people don't need to look it up... although it is probably displayed in the GCS

Reply:
done